### ISSUE TITLE
Events not firing

### ISSUE THREAD

**Issue by askz** [CroudSupport/vue-fullcalendar#11]
I think this  a duplicate of #8 but, there isn't real and clear solution proposed so I think this need more attention.

I've been testing many solutions but the basic example should work I suppose.
You'll find my code below. Events are been loaded but I can't do anything with them... Kind of frustrating
```
<template>
  <div class="hello">
    <full-calendar :eventSources="eventSources"> </full-calendar>
  </div>
</template>

<script>
import Vue from 'vue'
import FullCalendar from 'vue-full-calendar'
window.jQuery = window.$ = require('jquery')

// import hub from '../main'
// import 
import axios from 'axios'
var fcEvents = null

export default {
  name: 'hello',
  data () {
    return {
        eventSources: [
        {
          events(start, end, timezone, callback) {
            axios.get('http://localhost:5000/rendezvous/events/')
            .then(function(response){
                console.log(response.data)
                fcEvents = response.data
                callback(response.data)
            })
          }
        }
      ]
    }
  },
  ready(){
    this.$on('event-resize', console.log('help'))
  },
  events: {
    'event-drop': function(event){
      console.log(event)
    }
  },
  methods: {
    'eventResize' (event) {
      console.log(event)
    },
    'eventClick' (event, jsEvent, pos) {
      console.log('eventClick', event, jsEvent, pos)
    },
  }
}

</script>

<!-- Add "scoped" attribute to limit CSS to this component only -->
<style scoped>
h1, h2 {
  font-weight: normal;
}

ul {
  list-style-type: none;
  padding: 0;
}

li {
  display: inline-block;
  margin: 0 10px;
}

a {
  color: #42b983;
}
</style>
`

**Comment by neofreko** [comment]
@askz I had to made some adjustment for Vue2. I had similar issue. Then I change the event implementation to trigger Vue event instead of using jquery trigger. Something along:
```
eventClick(event, jsEvent, view) {
    self.$emit('event-selected', event, jsEvent, view)
},

eventDrop(event, delta, revertFunc, jsEvent, ui, view) {
    self.$emit('event-drop', event, delta, revertFunc, jsEvent, ui, view )
},
```

**Comment by askz** [comment]
Thanks @neofreko I ended up doing this too....

**Comment by arifhussain353** [comment]
`  <full-calendar :events="events"  :event-selected="eventClick"  :defaultView="defaultView" :header="header"></full-calendar>`

and js code look like this below : 

`   events: {
       'event-selected'(event) {
                console.log(event);
            }
         },
`

`    methods: {
          eventClick(event) {
                console.log(event)
            }
        }
`
Event is not triggering.I don't know where i am doing mistake

**Comment by BrockReece** [comment]
Hi everyone,

Sorry about the lack of activity on here recently. I have been pretty flat out and have only recently migrated to using Vue 2 and as such only just using v1 of this package in anger.

In v1.1.0 I updated how events are handled and I have updated the readme to reflect this.

I have also put together a quick example app to show most of this in action
https://github.com/BrockReece/vue-fullcalendar-example

I hope this helps.
Brock
